### PR TITLE
PTEMSVT-83 Rearange service questions and conditional render

### PR DESCRIPTION
**Pull request by ghost**
**Note**: Delete the description statements, complete each step. **None are optional**, but can be justified as to why they cannot be completed as written. Provide known gaps to testing that may raise the risk of merging to production.

## Are you removing, renaming or moving a folder in this PR?
- [x] No, I'm not changing any folders (skip to Summary and delete the rest of this section)
- [ ] Yes, I'm removing, renaming or moving a folder

Did you change site-wide styles, platform utilities or other infrastructure?
- [x] No
- [ ] Yes, and I used the [proxy-rewrite steps](https://github.com/department-of-veterans-affairs/vets-website/tree/main/src/applications/proxy-rewrite#that-sounds-normal-so-whats-the-proxy-all-about) to test the injected header scenario

## Summary

This PR changes the order of the Service page questions. It also adds a conditional  question based on whether or not the user answers yes or no to currently serving. If they "Yes" the question if they know when they will complete their service is rendered. If they choose "No" they skip a question go to the separation page.

 ## Related issue(s)

- _Link to ticket created in va.gov-team repo_
department-of-veterans-affairs/va.gov-team#0000
- _Link to previous change of the code/bug (if applicable)_
department-of-veterans-affairs/vets-website#0000
- _Link to epic if not included in ticket_
department-of-veterans-affairs/va.gov-team#0000

## Testing done

Manual testing

## Screenshots
Screen Shot 2024-08-26 at 12 48 09 PM
Screen Shot 2024-08-26 at 12 48 00 PM
Screen Shot 2024-08-26 at 12 47 50 PM

_Note: This field is mandatory for UI changes (non-component work should NOT have screenshots)._

|         | Before | After |
| ------- | ------ | ----- |
| Mobile  |        |       |
| Desktop |        |       |

## What areas of the site does it impact?

*(Describe what parts of the site are impacted **if** code touched other areas)*

### Quality Assurance & Testing

- [x] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [x] Linting warnings have been addressed
- [ ] Documentation has been updated ([link to documentation](#) \*if necessary)
- [ ] Screenshot of the developed feature is added
- [x] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [x] Browser console contains no warnings or errors.
- [ ] Events are being sent to the appropriate logging solution
- [ ] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [ ] Did you login to a local build and verify all authenticated routes work as expected with a test user

## Requested Feedback

(OPTIONAL) _What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_

### CODE REVIEW

**Review by vetskrieg — DISMISSED** [review]

File: src/applications/benefit-eligibility-questionnaire/pages/militaryService.js [hunk 1]
```diff
@@ -20,6 +20,19 @@ export default {
         },
       },
     },
+    expectedSeparation: {
+      'ui:title': (
+        <>
+          <p>
+            <b>When do you expect to seperate or retire from the service?</b>
```
Comment (jamigibbs) [inline src/applications/benefit-eligibility-questionnaire/pages/militaryService.js:27]: Should "seperate" be spelled "separate"?

File: src/applications/benefit-eligibility-questionnaire/pages/militaryService.js [hunk 2]
```diff
@@ -28,6 +41,16 @@ export default {
         type: 'string',
         enum: ['Yes', 'No'],
       },
+      expectedSeparation: {
+        type: 'string',
+        enum: [
+          'Within the next 3 months',
+          'More than 3 months but less than 6 months',
+          'Within more than 6 months but less than 1 year',
```
Comment (jamigibbs) [inline src/applications/benefit-eligibility-questionnaire/pages/militaryService.js:49]: Should this sentence instead be "More than 6 months but less than 1 year"?

**Review by jamigibbs — DISMISSED** [review]

File: src/applications/benefit-eligibility-questionnaire/pages/militaryService.js (hunk 1 above)
Comment (iseabock-va) [reply src/applications/benefit-eligibility-questionnaire/pages/militaryService.js]: Good catch! Thanks

File: src/applications/benefit-eligibility-questionnaire/pages/militaryService.js (hunk 2 above)
Comment (iseabock-va) [reply src/applications/benefit-eligibility-questionnaire/pages/militaryService.js]: Good catch! Thanks

**Review by vetskrieg — APPROVED** [review]

**Review by jamigibbs — APPROVED** [review]

**Pull request merged**